eventbridge-sqs-ecs-cdk/README.md

```diff
@@ -0,0 +1,63 @@
+# EventBridge to SQS to ECS
+
+This project contains sample AWS CDK code to integrate Amazon EventBridge, Amazon SQS and Amazon Elastic Container Service (ECS). As a new event is sent to EventBridge, it is queued in SQS and based on the number of messages that queue has, it creates an ECS cluster with Fargate task definitions. 
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/eventbridge-sqs-ecs-cdk
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the AWS Pricing page (https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+
+## Requirements
+
+* Create an AWS account (https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* AWS CLI (https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* Git Installed (https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* Node and NPM (https://nodejs.org/en/download/) installed
+* Python, pip (https://docs.aws.amazon.com/cdk/latest/guide/work-with-cdk-python.html) installed
+* AWS Cloud Development Kit (https://docs.aws.amazon.com/cdk/latest/guide/cli.html) (AWS CDK) installed
+
+## Deployment Instructions
+
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd eventbridge-sqs-ecs-cdk/src
+    ```
+3. Install the project dependencies
+    ```
+    python -m pip install -r python-app/requirements.txt
+    ```
+4. Deploy the stack to your default AWS account and region.
+    ```
+    cdk deploy
```

Comment:
This is a node cdk project, so the instructions are missing how to install the node dependencies, eg npm i before the cdk deploy. 
Also when trying to deploy this there are TS errors. Please fix the project and the instructions to for a person that downloads this project from GithHub.

Reply:
Hi, I have updated the file to include npm install command. However, I don't see any TS error at my end. Can you please provide more details on it?

Thank you.